gitpractice/javascript/task9.js

```diff
@@ -0,0 +1,49 @@
+//http://www.codewars.com/kata/prefill-an-array
+function prefill(number, v) {
+    var array = [];
+    if (number === 0 || number==="0")
```

Comment:
i think you can put all the conditions in one single if condition.
try to do it in that way.